.gitignore

```diff
@@ -1,7 +1,2 @@
 *~
-.settings
```

Comment:
I checked the project with SensioLabsInsight (https://insight.sensiolabs.com/) for coding style issues. They complained .idea, .project and .buildpath are files from an IDE and should be part of an projects .gitignore file. For this a deleoper should use the global setting from git.

If you would to have them in the .gitignore file, I will add them again.